packages/block-library/src/column/save.js

```diff
@@ -18,8 +18,18 @@ export default function save( { attributes } ) {
 	let style;
 
 	if ( width ) {
+		let flexBasis = Number.isFinite( width ) ? width + '%' : width;
+		// In some cases we need to round the width to a shorter float.
+		if ( ! Number.isFinite( width ) && width?.endsWith( '%' ) ) {
+			const multiplier = 1000000000000;
+			// Shrink the number back to a reasonable float.
+			flexBasis =
+				Math.round( Number.parseFloat( width ) * multiplier ) /
+					multiplier +
+				'%';
+		}
 		// Numbers are handled for backward compatibility as they can be still provided with templates.
```

Comment:
Do we want to move this comment above line 21 since the operation it's related to has been relocated?

Reply:
This should be fixed now.